core/client/src/client.rs

```diff
@@ -1083,21 +1089,36 @@ impl<B, E, Block> api::BlockBuilder<Block> for Client<B, E, Block> where
 	Block: BlockT,
 {
 	type Error = Error;
+	type OverlayedChanges = OverlayedChanges;
 
-	fn initialise_block(&self, at: &BlockId<Block>, header: <Block as BlockT>::Header) -> Result<(), Self::Error> {
-		self.call_api_at(at, "initialise_block", &(header))
+	fn initialise_block(
+		&self,
+		at: &BlockId<Block>,
+		changes: &mut OverlayedChanges,
+		header: &<Block as BlockT>::Header
+	) -> Result<(), Self::Error> {
+		self.call_at_state(at, "initialise_block", header, changes)
```

Comment:
should this code block be autogenerated from the API?

Reply:
I thought about it. But did not had any good idea yet.